pkg/metainfo/kvmetainfo/buckets.go

```diff
@@ -17,8 +17,19 @@ func (db *Project) CreateBucket(ctx context.Context, bucketName string, info *st
 	if bucketName == "" {
 		return storj.Bucket{}, storj.ErrNoBucket.New("")
 	}
+	if info == nil {
+		info = &storj.Bucket{PathCipher: storj.AESGCM}
+	}
+	if info.EncryptionParameters.CipherSuite == storj.EncUnspecified {
+		info.EncryptionParameters.CipherSuite = storj.EncAESGCM
+	}
```

Comment:
How about all the rest of the `storj.Bucket` fields? Shouldn't we set them to some sane default values if `info == nil` or have zero values to in `info`?

Reply:
It looked to me like zero values in all the other positions were expected or supported. Any fields in particular?

Reply:
We should check and set defaults at least for the redundancy scheme. Otherwise, a zero redundancy scheme will be stored in the bucket metadata.

Reply:
Actually, I've just checked that we actually need all encrypted block size, redundancy scheme, and segments size set properly here.

Reply:
I opened #1710 to be merged to this PR with suggestion how to set correct default for bucket metadata.

Reply:
For my own curiosity, why does a bucket need to have a nonzero redundancy strategy?

Reply:
`libuplink.Project.OpenBucket` initializes lots of things (like redundancy strategy) based on the bucket config we store in the bucket metadata. So we have to make sure that `kvmetainfo.Project.CreateBucket` stores good defaults in the bucket metadata if the respective config is not provided by the caller.